Compute/PollingAgent/src/Program.cs

```diff
@@ -199,22 +198,9 @@ public static async Task<int> Main(string[] args)
                           });
 
       app.MapGet("/taskprocessing",
-                 async () =>
-                 {
-                   var pollster  = app.Services.GetRequiredService<Common.Pollster.Pollster>();
-                   var exManager = app.Services.GetRequiredService<ExceptionManager>();
-
-                   var check = await pollster.Check(HealthCheckTag.Liveness)
-                                             .ConfigureAwait(false);
```

Comment:
why is this healthcheck no longer needed ?

Reply:
Because we recheck later if the pod is still there and processing the task or not (when the message is pulled again on another pod).